oneflow/core/functional/impl/array_functor.cpp

```diff
@@ -283,16 +285,36 @@ class EmptyFunctor {
                       device_symbol->device_id());
     if (device.has_value()) {
       Symbol<Device> device_symbol = JUST(device);
-      return OpInterpUtil::Dispatch<Tensor>(*op_, {}, OpExprInterpContext(attrs, device_symbol));
+      empty =
+          JUST(OpInterpUtil::Dispatch<Tensor>(*op_, {}, OpExprInterpContext(attrs, device_symbol)));
     } else {
-      return OpInterpUtil::Dispatch<Tensor>(*op_, {}, attrs);
+      empty = JUST(OpInterpUtil::Dispatch<Tensor>(*op_, {}, attrs));
     }
+
+    if (dtype->is_floating_point()) { JUST(empty->set_requires_grad(requires_grad)); }
+    return empty;
   }
 
  private:
   std::shared_ptr<OpExpr> op_;
 };
 
+class EmptyStridedFunctor {
+ public:
+  Maybe<Tensor> operator()(const std::vector<int64_t>& shape, const std::vector<int64_t>& stride,
+                           const Optional<Symbol<DType>>& dtype,
+                           const Optional<Symbol<Device>>& device, const bool requires_grad,
+                           const bool pin_memory) const {
+    Symbol<DType> data_type = GetDefaultDType();
+    if (dtype.has_value()) { data_type = JUST(dtype); }
+    auto empty =
+        JUST(functional::Empty(Shape(shape), data_type, device, requires_grad, pin_memory));
```

Comment:
```suggestion
        JUST(functional::Empty(Shape(shape), dtype.value_or(GetDefaultDType()), device, requires_grad, pin_memory));
```
data_type 和 dtype 是两种类型，感觉这里这么改容易混淆